test/.jshintrc

```diff
@@ -3,7 +3,7 @@
 	"qunit": true,
 
 	"-W053": true,
-	"-W097": true, // disable warning for 'Use the function form of "use strict".'
```

Comment:
We can also create a new PR that adds the comments in all cryptic parameters depending on the response of https://github.com/jshint/jshint/issues/741#issuecomment-268786491